### ISSUE TITLE
Add App to Package Manager?

### ISSUE THREAD

**Issue by Adhjie** [mjishnu/alt-app-installer#65]
Add App to Package Manager (e.g., preferably Chocolatey, Winget)?

Should I make 2 issue here or is this one just about UI?
2.
> You can also install already downloaded apps via "Install from file" in "Options"

I only see "Install UWP App" option here, is this for all apps or just UWP. Also does it also fetch all the latest (Retail label in https://store.rg-adguard.net ) dependencies?

Does this have the same mechanism as installing apps from downloaded files from https://store.rg-adguard.net and then using Add-AppxPackage to install it?

Edit: Spellings

**Comment by mjishnu** [comment]
1. that's a great idea I will do it.
2. it's to install uwp apps, the wording changed with updates the i forgot to update it in readme
3. yes it does, install uwp run the command for you along the path you selected though this doesn't use store.rg-adguard it directly fetches the file from MS servers

**Comment by Adhjie** [comment]
Forgot to add, addendum:
Just use Winget, Chocolatey package will be broken if the repository has a URL path change. This is not shown in the website of 'choco install <package>' but is visible when searched in CLI: e.g., Sysinternals by Microsoft is broken for FOSS user, its cached for licensed user. usually because of path change. Nirsoft still works but the maintainer is not active currently.

Technically, there are most package managers (excluding msys2 (not really a package manager, but there's a release of czkawka; krokiet,etc. there, as an alternative to the github version), cygwin, that I know of) inside UniGetUI, also python. so not sure if its doable, just the installer or portable version using 'pip'?

Any choice works really, though I really like Numstr take on Obsidian Portable.

And lastly, just pick the choice that is not a burden to maintain, portability is 2nd. Complex code or, pardon the word, 'spaghetti code, would be a dead end (for most small FLOSS dev) (either dead end, or entering maintenance mode, e.g., SD Maid. the rewrite is SD Maid SE. I'm also afraid if syncthing-Fork would stop, and will be the end for my android storage space).
Unless a rewrite is possible, IG (hypothetical full backup from Tachiyomi to the successor [not possible]).

tl;dr
My current wish would be unofficial PortableApps form with self-extractor, e.g., how Numstr/Obsidian Portable did it (unofficial take, since the self-extraction is diff from official, but better since it's DIY).
2nd would be just winget, doable in UniGetUI in the 'custom arguments: bar' in the case winget error or not installed in the user machine.

P.S.
Is it possible to install system app (e.g., https://apps.microsoft.com/detail/9nblggh4nns1 link from https://github.com/microsoft/winget-cli ) using alt-app-installer (winget can't install its own desktop app installer)?
I broke my winget with installing preview, currently waiting until preview version is older than latest stable release, then install it using Add-AppxPackage https://store.rg-adguard.net/ .appx

**Comment by mjishnu** [comment]
i was planning to add it to Chocolatey but i guess Winget should also do the job, the reason it is delayed is due to the fact that the app is is kinda broken due to microsoft site recent changes (#66 ), so waiting untill a new version for pyqt webengine with fix is released.

regarding ur second query i am not sure i heard you can install microsft app installer using winget but never really tried u can try using alt-app-installer get from link option it should work fine since its a uwp app

**Comment by mjishnu** [comment]
i know its been 2 years lol but its finally added to winget you can check it out using this

```ps
winget install mjishnu.altappinstaller
```

**Comment by Adhjie** [comment]
Okay. Thanks for the update.
